rmw_connext_cpp/CMakeLists.txt

```diff
@@ -2,7 +2,12 @@ cmake_minimum_required(VERSION 3.5)
 
 project(rmw_connext_cpp)
 
-set(CONNEXT_STATIC_DISABLE $ENV{CONNEXT_STATIC_DISABLE}
+if(DEFINED ENV{CONNEXT_STATIC_DISABLE})
+  set(disable $ENV{CONNEXT_STATIC_DISABLE})
```

Comment:
@dirk-thomas nit: can we replace generic `disable` variable name with `_connext_static_disable` or `disable_static_connext` ?

Reply:
Done: f8690e8f88f39c8a0d63f920f8fd0ee3d26fce52.